src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -817,6 +821,24 @@ public final void testPostGobiOrdersGetExistingPOLineNumberFails() throws Except
     logger.info("End: Testing for 201 - Create new Order, if fetching existing PO Line Number Fails");
   }
 
+  @Test
+  public final void testPostElectronicMonographISBNQualifier(TestContext context) throws Exception {
+    logger.info("Begin: Testing for 201 - XML content");
+
+    final Async asyncLocal = context.async();
+
+    final String body = getMockData(PO_LISTED_ELECTRONIC_MONOGRAPH_QUALIFIER_PATH);
+    postOrderSuccess(body);
+
+    Map<String, List<JsonObject>> postOrder = MockServer.serverRqRs.column(HttpMethod.POST);
+    CompositePurchaseOrder compPO = postOrder.get(PURCHASE_ORDER).get(0).mapTo(CompositePurchaseOrder.class);
+    assertNotNull(compPO.getCompositePoLines().get(0).getDetails().getProductIds().get(0).getQualifier());
+    assertEquals("9781410352224",compPO.getCompositePoLines().get(0).getDetails().getProductIds().get(0).getProductId());
```

Comment:
It seems that the value of the qualifier is not checked anywhere.

Reply:
added test cases covering various scenarios